data/docs/useSetState.md

```diff
@@ -0,0 +1,19 @@
+---
+id: useSetState
+title: useSetState
+sidebar_label: useSetState
+---
+
+## About
+
+Manage the state of a Set in React. Exposes add, delete and clear methods along with the current set to easily manipulate values in a set.
+
+## Examples
```

Comment:
Will add examples after it gets published to npm. Because sandpack requires the package to be on npm.